schema/omega-services.async.api.yaml

```diff
@@ -0,0 +1,442 @@
+      
+    editSetSummary:
+      type: object
+      properties:
+        oci:
+          $ref: "#/components/schemas/omegaCatalogueIdentifier"
+        version:
+          $ref: "#/components/schemas/assetVersion"
+        ownerId:
+          $ref: "#/components/schemas/assetOwnerId"
+        title:
+          $ref: "#/components/schemas/assetTitle"
+        description:
+          $ref: "#/components/schemas/assetDescription"
+        members:
+          type: array
+          items:
+            $ref: '#/components/schemas/editSetSummaryMember'
+
+    editSetSummaryMember:
+      type: object
+      description: The Summary of an EditSet member
+      properties:
+        oci:
+          $ref: "#/components/schemas/omegaCatalogueIdentifier"
+        version:
+          $ref: "#/components/schemas/assetVersion"
+        ownerId:
+          $ref: "#/components/schemas/assetOwnerId"
+        ccr:
+          $ref: "#/components/schemas/classicCatalogueReference"
+        title:
+          $ref: "#/components/schemas/assetTitle"
+        description:
+          $ref: "#/components/schemas/assetDescription"
+        coveringDate:
+          type: string
+          description: The covering dates of the EditSet Member
+
+    editSetRecord:
+      type: object
+      description: The properties of a record in an edit set.
+      required:
+        - ccr
+        - oci
+        - description
+        - coveringDate
+        - formerReferenceDepartment
+        - formerReferencePro
+        - startDateDay
+        - startDateMonth
+        - startDateYear
+        - endDateDay
+        - endDateMonth
+        - endDateYear
+        - legalStatusID
+        - placeOfDepositID
+        - note
+        - background
+        - custodialHistory
+        - relatedMaterial
+        - separatedMaterial
+        - creatorIDs
+
+      properties:
+        ccr:
+          description: |
+            CCR stands for "Classic Catalogue Reference"
+            The unique identifier which links the catalogue to the records it
+            represents and allows the user to order and refer to them.
+          $ref: "#/components/schemas/classicCatalogueReference"
+        oci:
+          $ref: "#/components/schemas/omegaCatalogueIdentifier"
+        description:
+          type: string
+          description: |
+            A prose statement giving information about the form and subject
+            matter of the unit of description (often in terms of subjects,
+            people and places). Its purpose is to enable users to judge the
+            potential relevance of the record described and whether they want
+            to investigate the material further or not.
+
+            Note: Called "Scope and Content" in TNA-CS13
+        coveringDate:
+          type: string
+          description: |
+            Identifies and records the date(s) of creation of the records being
+            described.
```

Comment:
I agree with your comment in the ticket that this should be `coveringDates` as this is the term used in CS13 (catalogue standards document).

Reply:
Changed to `coveringDates`, in both the record and the summary.